kiali-server/values.yaml

```
@@ -73,6 +73,11 @@ identity: {}
   #private_key_file:
 
 kiali_feature_flags:
+  certificates_information_indicators:
+   enabled: true
+   secrets:
+   - istio-ca-secret
+   - cacerts
```

can you swap this to sort them alphabetically?
It's stupid, I know - but when we do diff's on these lists, its nice when things are sorted because the diffs won't just show different orders of things.
so this is trivial:
```suggestion
   - cacerts
   - istio-ca-secret
```